includes/rest-api/class-sensei-rest-api-plugins-controller.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Sensei REST API: Sensei_REST_API_Plugins_Controller class.
+ *
+ * @package sensei-lms
+ * @since   3.11.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+} // Exit if accessed directly.
+
+/**
+ * A REST controller for Sensei related plugins.
+ *
+ * @since 3.11.0
+ *
+ * @see   WP_REST_Controller
+ */
+class Sensei_REST_API_Plugins_Controller extends WP_REST_Controller {
+	/**
+	 * Routes namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace;
+
+	/**
+	 * Routes prefix.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'sensei-plugins';
```

Comment:
Not a big deal. Just wondering if we should call it as `sensei-extensions` if we want also add themes in the future. And also to match the menu name, and so on. I'm deducing it because of the current implementation we have in the `get_extensions` method. But I'm really not sure. :P

Reply:
I was thinking about that too: 840e6638a076987ca70c1284b3fb243e83794ece

Reply:
As we made the endpoint more general, it made sense to add a type argument: 81dbc33c9182d2b380c932d47498df2801e47c1a

Reply:
Makes sense! Good idea!